lib/uppy/ping/scheme.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Uppy.Ping do
+  use Ecto.Schema
+
+  alias Uppy.Job
+
+  @primary_key {:id, :binary_id, autogenerate: true}
+  @foreign_key_type :binary_id
+
+  schema "pings" do
+    field :response, :string
+    field :time, :integer
+
+    belongs_to :job, Job
+
+    timestamps()
```

Comment:
If this is not meant to be updated, we can same just the inserted_at timestamp.

Reply:
[4f26776](https://github.com/andreicek/uppy/pull/2/commits/4f26776e08e2c02f021be8ec320c09cc18d0ac88)